<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveToLibrary</class>
 <widget class="QDialog" name="SaveToLibrary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>功能选择</string>
   </property>
   <widget class="QRadioButton" name="m_radio1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>选择区域作为图库</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_radio2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>从文件导入图片</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_transparentcheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>透明</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_lable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>透明色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_color">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_radio3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>导入图库</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="MainView">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>551</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_SaveTo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>398</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存到...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_Cancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HelpButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
